cli/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "claimable-tokens-cli"
+version = "0.1.0"
+authors = ["Atticlab team"]
```

Comment:
You can keep yourself as the author, nothing wrong with some self-promotion. :)
If you use Attic, then it is better to write `Attic Lab`.